src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/CancellationsListResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CancellationsListResponse : Identifiable, ICancellationsListResponse
+    {
+        public CancellationsListResponse(Uri id, List<ITransaction> cancellationList)

```

Comment:
`id` can just be passed to `base(id)`, no? Then the setter of the `Id` property can be removed.